src/Microsoft.VisualStudio.Validation/Requires.cs

```diff
@@ -555,7 +555,7 @@ public static void NotDefault<T>(T value, [CallerArgumentExpression(nameof(value
         var defaultValue = default(T);
         if (defaultValue.Equals(value))
         {
-            throw new ArgumentException(PrivateErrorHelpers.Format(Strings.Argument_StructIsDefault, parameterName, typeof(T).FullName), parameterName);
+            throw new ArgumentException(Strings.FormatArgument_StructIsDefault(parameterName, typeof(T).FullName!), parameterName);
```

Comment:
📝 The arguments are now marked nullable, so this suppression should be unnecessary.

```suggestion
            throw new ArgumentException(Strings.FormatArgument_StructIsDefault(parameterName, typeof(T).FullName), parameterName);
```

Reply:
Nice catch. I thought I'd removed them all but I missed that one.